stripe should not be a devDependency of auth-server

https://github.com/mozilla/fxa/blob/2e692e9fa13f30e016e1c514ab587e56a7c66ef6/packages/fxa-auth-server/package.json#L108
https://github.com/mozilla/fxa/blob/2e692e9fa13f30e016e1c514ab587e56a7c66ef6/packages/fxa-auth-server/package.json#L141

Those lines are in devDependencies, but the module is loaded at startup.

`docker run -it --rm -e NODE_ENV=dev mozilla/fxa-auth-server:latest node bin/key_server.js` fails with `Error: Cannot find module 'stripe'`.

--- a/packages/fxa-auth-server/package.json
+++ b/packages/fxa-auth-server/package.json
@@ -88,6 +88,7 @@
     "safe-regex": "1.1.0",
     "safe-url-assembler": "1.3.5",
     "sandbox": "0.8.6",
+    "stripe": "^7.13.1",
     "through": "2.3.8",
     "urijs": "1.19.1",
     "uuid": "1.4.1",
@@ -138,7 +139,6 @@
     "simplesmtp": "0.3.35",
     "sinon": "7.0.0",
     "sjcl": "1.0.6",
-    "stripe": "^7.13.0",
     "typescript": "^3.6.4",
     "ws": "5.2.2"
   }

